src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -21,16 +27,58 @@ export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
 
   return (
     <article className={styles.SingleArticle}>
-      <div className={styles.title}>
+      <div>
```

Comment:
Q: Where do ORBIT Blog articles fit into this? Is that considered internal news? I know there are two separate categories in the CMS for internal news vs orbit blog, but don't see ORBIT Blog reflected in the constants in this repo. Did we miss something in the past, or am I getting the information architecture confused?

Reply:
Great question. I think we maybe missed this in the past? I think I added in the constants when creating the internal news page since we were trying to differentiate between what is public-facing and what isn't. But I don't believe the ORBIT Blog was available at that point? Since the blog is just updates that we are making, and the portal is open-source, it doesn't seem like it should be considered "internal." I guess this would ultimately be a design decision as to whether or not we need a new tag for this? Maybe we can discuss in our sync today.